### ISSUE TITLE
Enhancement | Download Tools Manager

### ISSUE THREAD

**Issue by Danik2343** [Kiki79250CoC/SMWMTL#8]
A new suggestion by  for a new app / update app version. 

App name: SMW MultiTools Launcher
App version: 2.28s
Newer version: Yes
Optional message: 

Please add ability to download Applications and manager their versions for Launcher
Lunar Magic and e.t.c

-------------------------------------------------------------------------------------------------------

SMWMTL_Montréal_2.28.1610.81-rv8bld-01_x64_240923

**Comment by Kiki79250CoC** [comment]
Yeah, this is something I could consider for the unrestricted build, but I fear this will not be possible for the SMWC-restricted one, as I was constrained to remove all network-related features (like the auto-updater) to get SMWMTL accepted into SMWC (that's the reason to the presence of the "s" in version number, it means "stripped down" (you can call it "sabotaged" if you want, as it is somewhat valid).).

Probably I will have to talk with them before consider adding a such feature, in order to see and delimit what I can or can't do.

I will see for it later, at the moment I'm still under making the French Lunar Magic translation, and considering the speed I progress (I'm a bit of perfectionist, so it takes a bit long just to translates 10k strings), let's hope I can get some discussion around the end of this year/begin of 2024.

I'll put your suggestion in the "wait and see" box for now, I will certainly go back and do some sketches and prototypes in the future if I and SMWC can get to something conclusive.

**Comment by Danik2343** [comment]
> Yeah, this is something I could consider for the unrestricted build, but I fear this will not be possible for the SMWC-restricted one, as I was constrained to remove all network-related features (like the auto-updater) to get SMWMTL accepted into SMWC (that's the reason to the presence of the "s" in version number, it means "stripped down" (you can call it "sabotaged" if you want, as it is somewhat valid).).
> 
> Probably I will have to talk with them before consider adding a such feature, in order to see and delimit what I can or can't do.
> 
> I will see for it later, at the moment I'm still under making the French Lunar Magic translation, and considering the speed I progress (I'm a bit of perfectionist, so it takes a bit long just to translates 10k strings), let's hope I can get some discussion around the end of this year/begin of 2024.
> 
> I'll put your suggestion in the "wait and see" box for now, I will certainly go back and do some sketches and prototypes in the future if I and SMWC can get to something conclusive.

Then this functionality can be added to the version without "S", I installed the version without "S"
